streams/src/test/java/org/apache/kafka/streams/processor/internals/InternalTopicManagerTest.java

```diff
@@ -153,9 +153,9 @@ public void shouldNotCreateTopicsWithEmptyInput() throws Exception {
 
     @Test
     public void shouldOnlyRetryNotSuccessfulFuturesDuringSetup() {
-        final AdminClient admin = EasyMock.createNiceMock(AdminClient.class);
+        final AdminClient admin = EasyMock.createMock(AdminClient.class);
```

Comment:
Is this a necessary change? If so, should we change all similar code in this test class?

Reply:
See https://github.com/apache/kafka/pull/10529#discussion_r612291115

Reply:
@cadonna thanks!